odl/tomo/geometry/conebeam.py

```diff
@@ -24,7 +25,8 @@
 
 
 __all__ = ('FanFlatGeometry',
-           'CircularConeFlatGeometry', 'HelicalConeFlatGeometry',)
+           'CircularConeFlatGeometry', 'HelicalConeFlatGeometry',
```

Comment:
I vote for removing `CircularConeFlatGeometry` now that we have the utility. Advanced users should know to use the helical geometry.